### PR TITLE
Merge pull request #2 from btel/fix-readme

### LINKED ISSUE #1: Improvements to make in this repository
In https://github.com/jupyterhub/binder/pull/181 we identified some improvements that should be made to this repository. This issue is a quick list so that we can work on them.

@btel it would be fantastic if you could give this a shot :-)

* Remove language that is for Bokeh (e.g. https://github.com/binder-examples/tensorboard/blob/master/tensorboardserverextension.py#L5)
* Update the README so that it is consistent with the other `binder-examples` repositories. E.g., that the headers are the same etc (see https://github.com/binder-examples/conda for example)

### PR DESCRIPTION
**Pull request by btel**
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,40 @@
...
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/binder-examples/tensorboard/master?filepath=train_model.ipynb) (notebook)
 
 An example of running Jupyter notebook + tensorboard on mybinder.org. To launch tensorboard go to:
 
 https://mybinder.org/v2/gh/btel/binder-tensorboard/master?urlpath=%2fproxy%2f6006%2f
 
-Once the tensorboard is launched you can also see the notebooks by replacing the /proxy/6006/ part of the URL (or everything after `/proxy`) with `/tree` (or `/lab` for jupyter lab view).
+This example shows how to:
+
+* expose [tensorboard](https://www.tensorflow.org/tensorboard/) interface on
```
Comment (choldgraf) [inline README.md]:
```suggestion
* expose the [tensorboard](https://www.tensorflow.org/tensorboard/) interface on
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
+  binder,
+* use
```
Comment (choldgraf) [inline README.md]:
```suggestion
* use the
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
+  [jupyter-server-proxy](https://github.com/jupyterhub/jupyter-server-proxy)
+  extension to map a web application running on a port inaccessible from the
+  internet (6006 for example) to jupyter path (`/proxy/6006/`),
+* launch tensorboard on startup using custom server extension
```
Comment (choldgraf) [inline README.md]:
```suggestion
* launch tensorboard on startup using the custom server extension
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,40 @@
...
 
-When you run the notebook you can see the progress of the run live! You can watch in a separate browser tab.
+When you run the training of the neural net in the notebook you can watch the
+progress of the run live (you can keep tensorboard open in a separate tab). 
 
 ## How?
 
-tensorboard launches a webserver on a given port (by default 6006). In many environments (like dokerized deployments), the port is not visible in the external network. To expose it, we use the jupyter-proxy-extension to proxy it to the jupyter server url (`/proxy/6006`)
+tensorboard launches a webserver on a given port (by default 6006). In many
+environments (like dokerized deployments), the port is not visible in the
```
Comment (choldgraf) [inline README.md]:
```suggestion
environments (like dockerized deployments), the port is not visible in the
```

**Review by choldgraf — CHANGES_REQUESTED** [review]
A few small grammar nitpicks in there, but other than that it looks good to me. Let me know when you've got those addressed!

**Comment by btel** [comment]
Perfect! Thanks for the corrections!

**Comment by choldgraf** [comment]
great! thanks so much for the improvements :-)

**Pull request merged**